### PR TITLE
Add the phylorthology test data

### LINKED ISSUE #15: Move the test files to Arcadia's test data repo
### Description of feature

https://github.com/Arcadia-Science/test-datasets

### PR DESCRIPTION
**Pull request by mertcelebi**
Fixes https://github.com/Arcadia-Science/phylorthology/issues/15

### CODE REVIEW

**Review by taylorreiter — APPROVED** [review]
LGTM, but should the fasta files be gzipped and the sample sheet paths updated, or will that break the phylorotholgy pipeline?

**Comment by mertcelebi** [comment]
I'll let @austinhpatton answer that. Currently, that'd break the pipeline I think as I don't know if orthofinder also works with zipped files.

File: phylorthology/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+## Information & purpose
+
+This repository holds the test dataset and the input file for the [phylorthology pipeline](https://github.com/Arcadia-Science/phylorthology).
+
+The [samplesheet CSV](samplesheet_test.csv) includes all the file paths and the metadata for the test proteomes. The [nextflow_parameters.json](nextflow_parameters.json) file specifies all the inputs to the [phylorthology pipeline](https://github.com/Arcadia-Science/phylorthology) and can be used via the `-params-file` CLI option.
+
+## Test dataset curation
+
+TODO: [Austin Patton](https://github.com/austinhpatton) will include more details around the curation of this dataset.
```
Comment (austinhpatton) [inline phylorthology/README.md:11]: I'm going to leave this as a TODO for now - we can figure this out in more detail once we work out what size of a dataset we should include as a test dataset (and how rigorous the results of that test should be).

File: phylorthology/samplesheet_test.csv [hunk 2]
```diff
@@ -0,0 +1,7 @@
+species,file,taxonomy,shallow,broad,mode,uniprot,mcl_test
+Entamoeba_histolytica,https://github.com/Arcadia-Science/test-datasets/raw/main/phylorthology/Entamoeba_histolytica-test-proteome.fasta,Amoebozoa,eukaryota_odb10,eukaryota_odb10,proteins,true,false
```
Comment (austinhpatton) [inline phylorthology/samplesheet_test.csv:2]: I'm a bit confused - why do we have the test data (fasta files) in the phylorthology repository if we're just specifying that we should pull the test data from the arcadia-science test datasets repository?

**Review by austinhpatton — APPROVED** [review]
Couple small comments/questions, but all looks good!

**Comment by austinhpatton** [comment]
Also, just seeing the comment about gzipping the fasta files - that should probably be something that we build into the workflow moving forward (capability of handling either gzipped or non-gzipped inputs). I don't gzip them because it just adds another layer of complexity, and the proteome files aren't _that_ large, so it's generally not even that much of a space saver. But, no point in forcing users to have to keep uncompressed inputs.

**Pull request merged**